### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by KathrinLuecke**
Corrected qr_code_scan_body_subtitle (https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10867)

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -920,7 +920,7 @@ Sollten Sie den Test in der App gelöscht haben, können Sie ihn aus dem Papierk
     <!-- YTXT: instruction text for QR code scanning  -->
     <string name="qr_code_scan_body_title">"Welche QR-Codes können Sie scannen?"</string>
     <!-- YTXT: instruction text for QR code scanning  -->
-    <string name="qr_code_scan_body_subtitle">"Ihre PCR-Tests, Schnelltests, Testzertifikate, Impfzertifikate, Genesenenzertifikate sowie Check-ins."</string>
+    <string name="qr_code_scan_body_subtitle">"Tests, Zertifikate, Check-ins und Nachweise bei Ticketbuchungen"</string>
```
Comment (Ein-Tim) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:923]: Can we please still leave the "Ihre Zertifikate"? There are more and more reports that the UQS in the CWA is used to _verify_ certificates from others.

**Review by Ein-Tim — COMMENTED** [review]
One little comment (:

**Comment by KathrinLuecke** [comment]
Close this one and create new one

File: Corona-Warn-App/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (KathrinLuecke) [reply Corona-Warn-App/src/main/res/values-de/strings.xml:923]: Thanks, yes, I'll add that.

Comment (Ein-Tim) [reply Corona-Warn-App/src/main/res/values-de/strings.xml:923]: Thanks @KathrinLuecke!

**Pull request closed without merging**